driver/normalizer/normalizer.go

```diff
@@ -212,24 +237,32 @@ var Normalizers = []Mapping{
 		},
 	)),
 	MapSemantic("ImportDefaultSpecifier", uast.Alias{}, MapObj(
-		Obj{
-			"local": Var("local"),
+		Fields{
+			{Name: "local", Op: Var("local")},
+			//FIXME(bzz): save this once we agree how
+			{Name: "leadingComments", Drop: true, Op: Any()},
 		},
 		Obj{
 			"Name": Var("local"),
 			"Node": UASTType(uast.Identifier{}, Obj{
-				uast.KeyPos: AnyNode(nil),
+				uast.KeyPos: Any(),
 				"Name":      String("."), // TODO: scope
 			}),
 		},
 	)),
 	MapSemantic("FunctionDeclaration", uast.FunctionGroup{}, MapObj(
-		Obj{
-			"id":        Var("name"),
-			"generator": Var("gen"),   // FIXME: define channels in SDK? or return a function?
-			"async":     Var("async"), // TODO: async
-			"body":      Var("body"),
-			"params": Each("params", Cases("param_case",
+		Fields{
+			{Name: "id", Op: Var("name")},
+			{Name: "generator", Op: Var("gen")}, // FIXME: define channels in SDK? or return a function?
+			{Name: "async", Op: Var("async")},   // TODO: async
+			{Name: "body", Op: Var("body")},
+			//FIXME(bzz): map predicate properly
```

Comment:
Can you please describe briefly what it is (if you already know, of course).

Reply:
added in a836d36